blackbox/docs/configuration/node.txt

```diff
@@ -58,52 +59,21 @@ to differentiate nodes upon startup:
 
   Whether or not this node will store data.
 
-**node.client**
-  | *Default:* ``false``
-  | *Runtime:* ``no``
-
-  Shorthand for: ``node.data=false`` and ``node.master=false``.
-
-**node.local**
-  | *Default:* ``false``
-  | *Runtime:* ``no``
-
-  If set to ``true``, the node will use a *JVM-local* transport and discovery.
-  *Used primarily for testing purposes.*
-
-Examples
---------
-
-A node by default is eligible as *master* and contains *data*.
-
-Nodes that only contain *data* but cannot become *master* will mainly execute
-and respond to queries:
-
-.. code-block::
-   yaml
-
-   node:
-     data: true
-     master: false
-
-*Master-only*-nodes that do not contain *data* but are able to become *master*
-can be used to separate cluster-management loads from the query execution loads:
-
-.. code-block::
-   yaml
+Using different combinations of these two settings, you can create four
+different types of node. Each type of node is differentiate by what types of
+load it will handle.
 
-   node:
-     data: false
-     master: true
+The four types of node possible are:
 
-Nodes that do not contain *data* and are not eligible as *master* are called
-*client*-nodes. They can be used to separate request handling loads:
-
-.. code-block::
-   yaml
-
-   node:
-     client: true
++---------------+----------------------------+------------------------------+
+|               | **Master**                 | **No Master**                |
++---------------+----------------------------+------------------------------+
+| **Data**      | Handles all loads.         | Handles request handling and |
+|               |                            | query execution loads.       |
++---------------+----------------------------+------------------------------+
+| **No Data**   | Handles cluster management | Handles request handling     |
```

Comment:
Maybe we should clarify here a bit. Nodes marked as `node.master`  *can* do cluster state management, but only one node at a time is elected as *the* master.